Whiskey/Functions/Get-MSBuild.ps1

```diff
@@ -96,6 +96,14 @@ function Get-MSBuild
 
     foreach( $instance in (Get-VSSetupInstance) )
     {
+        # SSMS versions 21+ are now included when retrieving instances of Visual Studio related products
+        # (ie. Get-VSSetupInstance). However the shipped version of MSBuild does not contains the required imports to
+        # build C# code. Only use MSBuild from Visual Studio.
+        if ($instance.DisplayName -notmatch 'Visual Studio')

```

Comment:
Use the same logic as the NuGet.exe folks:: https://github.com/NuGet/NuGet.Client/pull/6530/files#diff-8ac450991c576742dcf0cc4e3bace8a6fc8011d3a0bc0a8c782f53176e18acd6R71

```powershell
if ($instance.InstallationName.StartsWith('VisualStudio', [StringComparison]::Ordinal))
```